### PR TITLE
Add a setting to enable GMON

### PR DESCRIPTION
**Pull request by mixern6**
### PURPOSE
Add a setting to enable GMON

### CODE REVIEW

**Review by bsavery — CHANGES_REQUESTED** [review]
See comment on explanation.

File: pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py [hunk 1]
```diff
@@ -502,6 +502,17 @@ def houdini_parm_name(name):
             }
         ]
     },
+    {
+        'name': 'GMON',
+        'settings': [
+            {
+                'name': 'core:useGmon',
+                'ui_name': 'Use GMON',
+                'help': 'Use GMON',
```
[the remark this answers is not in the document]
Comment (mixern6) [reply pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py]: Updated the explanation in help

File: pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py [hunk 2]
```diff
@@ -502,6 +502,17 @@ def houdini_parm_name(name):
             }
         ]
     },
+    {
+        'name': 'GMON',
+        'settings': [
+            {
+                'name': 'core:useGmon',
+                'ui_name': 'Use GMON',
+                'help': 'Enable fireflies suppression by using adaptive median of mean estimator',
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py:511]: Thanks for adding the explanation.
It should probably also mention this is a Hybrid Pro only feature.

Comment (mixern6) [reply pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py:511]: Updated

**Review by V-Gorash — APPROVED** [review]

**Review by bsavery — APPROVED** [review]

**Pull request merged**